js/ui/text_box/ui.text_editor.base.js

```diff
@@ -442,6 +446,26 @@ const TextEditorBase = Editor.inherit({
         this._input().prop('spellcheck', this.option('spellcheck'));
     },
 
+    _cleanLabelObservable: function() {
+        if(this._$labelObservableTarget) {
+            resizeObserverSingleton.unobserve(this._$labelObservableTarget);
+        }
+    },
+
+    _setLabelWidth: function() {
```

Comment:
This function is named as `_setLabelWidth` but it makes several things:
- sets label width
- updates `_$labelObservableTarget` cached value
- initialized resizeObserver subscription

I think we should redevide a responsibility.

Maybe `_updateLabelContainer` (or `_getLabelContainer`) and `_observeLabelResize(shouldObserver)`? This will also get rid of [this method](https://github.com/DevExpress/DevExtreme/pull/19461/files#diff-d5f8ba33de2fd615f123442b1375acee5e5112c3357d545e514ca1838796c197R449)